bokehjs/src/coffee/models/glyphs/image.coffee

```diff
@@ -7,6 +7,13 @@ p = require "../../core/properties"
 
 class ImageView extends Glyph.View
 
+  initialize: (options) ->
+    super(options)
+    @listenTo(@model.color_mapper, 'change', () ->
+      if @_image?
```

Comment:
The nonselection Image glyph doesn't have this._image or this.image_data set, so the `render` raises an exception. This is a non-ideal workaround to that.

It'd probably be better to resolve that issue, so i'll look into it.